bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ClientConfiguration.java

```diff
@@ -1126,6 +1127,30 @@ public ClientConfiguration setEnsemblePlacementPolicy(Class<? extends EnsemblePl
         return this;
     }
 
+    /**
+     * Get the threshold for the number of pending requests beyond which to reorder
+     * reads. If <= zero, this feature is turned off.
```

Comment:
Nit: use &lt; instead of '<'. This is HTML

Reply:
Thanks for pointing out. Fixed in rev 2.